spec/controllers/auth_controller_spec.rb

```diff
@@ -1,17 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe AuthController, type: :controller do
+  before(:each) do
+    request.env['omniauth.auth'] = auth_hash = OmniAuth::AuthHash.new({'extra' => { 'raw_info' =>{
```

Comment:
Is there a helper here that we could introduce and apply to all our controller tests?